traffic_ops/testing/api/v4/tc-fixtures.json

```
@@ -907,7 +907,7 @@
             "regexRemap": null,
             "regionalGeoBlocking": false,
             "remapText": null,
-            "routingName": "cdn",
+            "routingName": "ccr-ds1",
```

Why set it to an invalid value? What scenario are we trying to test?

---

That value is not invalid, what it was set to *before* was actually hiding an error. Changing this was part of the fix for #7094.

---

oh.. ok

---

No, it seems we don't